src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -65,6 +69,43 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
       return meUserInterestsLoader?.().then(({ body }) => body)
     },
   },
+  interestsConnection: {
+    type: UserInterestConnection,
+    args: pageable({}),
+    resolve: async (
```

Comment:
**suggestion:** It would be nice to test this resolver

Reply:
There's no test for the `CollectorProfile` resolver, I'd need to create it from scratch, so on it.

Reply:
is this comment related to [this point](https://github.com/artsy/metaphysics/pull/5594#discussion_r1504026387)? 😅

Reply:
yes 🙈